source/server/code/views/choose_ltf.erb

```diff
@@ -6,19 +6,32 @@ $(() => {
 
   $('.type').html(cd.urlParam('type'));
 
-  $('button.next').click(() => {
-    const name = encodeURIComponent(cd.selectedDisplayName);
-    cd.goto(`/creator/confirm?${cd.urlParams()}&language_name=${name}`);
+  $('button.ok').click(() => {
+      const name = encodeURIComponent(cd.selectedDisplayName);
+      $.ajax({
+          type: 'POST',
+          url: `/creator/confirm?${cd.urlParams()}&language_name=${name}`,
+          dataType: 'json',
+          async: false,
+          success: (response) => {
+              if (cd.urlParam('type') === 'group') {
+                  cd.goto(response.route);
+              } else {
+                  cd.windowOpen(response.route);
+                  $('.home-icon').trigger("click");
```

Comment:
Maybe this could be one function?
`cd.openThenGoHome(respons.route)`

`cd.openThenGoHome = (url) => {
  const windowIsOpened = window.open(url, '_blank');
  if (windowIsOpened) {
    windowIsOpened.focus();
    $('.home-icon').trigger("click");
  } else {
    alert('Please, allow popups for this website.');
  }
}
`

Reply:
I am sure there are lots of things that could be tidied up in creator. It definitely needs some attention.
At the moment creator is mostly UX but it also has some actual functionality. For example, it creates the manifest from the exercise_name and the language_start_point display_name, and then passes the manifest to saver when it creates a kata/group. As we briefly discussed, it would be better if UX pass the two strings directly to saver and saver created the manifest itself. At the moment I think it's best to not worry about these smaller tidy ups. Let's first get this PR over the line.